### DISCUSSION TITLE
How can I convert hex code to rgba code for model-viewer?

### DISCUSSION THREAD

**Discussion by FehmiCan16** [google/model-viewer#3652] [Q&A]
Hello there. I want to set the base color of the material. but I couldn't turn the color I wanted. for example (255, 215, 0, 1) how can I convert gold color for model-viewer?

**Comment by elalish** [comment]
Did you follow this [example](https://modelviewer.dev/examples/scenegraph/#changeColor)? Keep in mind that gold is metallic, so you'd want to change the `metalness` to 1 as well, and you'd probably want it shiny, so `roughness` 0 or 0.1.

**Reply by FehmiCan16** [reply]
I solved the problem. The color code range should be between 0 and 1. the material property was already metallic, the problem was caused by a small syntax error
